tools/proofers/previewControl.js

```diff
@@ -180,6 +189,15 @@ $( function() {
         setViewColors(outerPrev);
         viewMode = previewStyles.initialViewMode;
         $("#" + viewMode).prop("checked", true);
+        // check if MathJax already loaded. Will break if load more than once
+        if(previewStyles.allowMathPreview && (typeof(MathJax) === 'undefined')) {
```

Comment:
I did it like this to allow caching which is disabled by default with getScript(). Although it probably won't make much difference. Perhaps it would be better to use the simplest code rather an unnoticeable improvement in performance.